BGAnimations/ScreenGameplay underlay/PerPlayer/StepStatistics/TapNoteJudgments.lua

```diff
@@ -1,4 +1,6 @@
-local player = ...
+local args = ...
+local player = args[1]
+local has_labels = args[2]
```

Comment:
I think you could directly do
```
local player, has_labels = ...
```

Reply:
I think I need to do `unpack(...)` but yeah done!